packages/fe-fpm-writer/templates/filter/Controller.js

```diff
@@ -0,0 +1,21 @@
+sap.ui.define(["sap/ui/model/Filter", "sap/ui/model/FilterOperator"], function(Filter, FilterOperator) {
+	"use strict";
```

Comment:
It looks like there is a mix of tabs and spaces in all the templates in the module. For the templates in this PR, can you switch to spaces?
@tobiasqueck, @nlunets, @devinea, @Klaus-Keller: What is the guidance on whitespaces in the templates?

Reply:
I think the recommended ui5 rules are spaces but i don't know what you've applied so far

Reply:
Thanks @nlunets. I see that @AlinaGovoruhina has switched to spaces now. I will mark this as resolved.